extensions/amp-carousel/0.1/slides.js

```diff
@@ -246,7 +246,7 @@ export class AmpSlides extends BaseCarousel {
     this.updateInViewport(oldSlide, false);
     this.updateInViewport(newSlide, true);
     this.setControlsState();
-    this.schedulePause(oldSlide);
+    this.scheduleUnload(oldSlide);
```

Comment:
So, I'm aware that we are blocked with couple of components here for pause. But, we are also blocked with the same components outside of carousel. I.e. with or without carousel we need to be able to pause brightcove video after this PR. We are kind of regressing here. Any suggestions? A quick question - if we put `display:none` on the iframe - does it solve our problem of pausing for these components?